main.html

```diff
@@ -1,16 +1,17 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-<meta charset="UTF-8">
-<title>Page Title</title>
-<link rel="stylesheet" href="main.css">
-<link rel="stylesheet" href="normalize.css">
+    <meta charset="UTF-8">
+    <meta content='width=device-width, initial-scale=1.0, user-scalable=0' name='viewport'/>
+    <title>Page Title</title>
+    <link rel="preload" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.css">
+    <link rel="stylesheet" href="main.css">
 </head>
 <body>
     <header>
         <div class="header">
             <figure class="header__image">
-                <img src = "lyft_logo.png" alt=""/>
+                <img src = "lyft_logo.png" class ="header-source__image" alt=""/>
```

Comment:
1 - src = "url" => src="url"
2 - header-source__image should use `header` as a block, https://en.bem.info/methodology/

Reply:
+